cmd/user/internal/interfaces/http/router.go

```diff
@@ -39,6 +40,7 @@ func NewRouter(logger *log.Logger, repository user_persistence.UserRepository, c
 		http_middleware.HSTS(),
 		http_middleware.Metrics(),
 		http_middleware.LimitRequestBody(int64(10<<20)), // 10 MB is a lot of text.
+		http_form_middleware.FormJson(),
```

Comment:
can you add it to auth service as well ? not sure if we have router there

Reply:
yep we do, add it to
https://github.com/vardius/go-api-boilerplate/blob/3f0c9cc9621d5488df9b9bc665d853f39dd3ec79/cmd/auth/internal/interfaces/http/router.go#L28
and will merge it

Reply:
added

Reply:
https://github.com/vardius/go-api-boilerplate/pull/60/commits/834b5debf8c18b322e1c35af410ffb15131cad16 ok perfect